site/_includes/macros/cards/hero-card-wide.njk

```diff
@@ -8,6 +8,8 @@
       <h1 class="type--h1 color-{{color}}-darkest">
         {% if titleIcon %}
           <div class="project-icon gap-right-300">
+            <div class="project-icon__cover cover-last"></div>
```

Comment:
Let's add a comment here explaining it's a Safari workaound, with link to this PR and/or webkit bug